src/vlrdevapi/_series/info/parser.py

```
@@ -138,10 +137,20 @@ def parse_series_info(html: HTMLParser) -> SeriesInfo:
     if vs_section:
         notes = vs_section.css(".match-header-vs-note")
         if notes:
-            first_note = notes[0].text(strip=True).lower()
-            if first_note == "final":
+            first_note_raw = notes[0].text(strip=True)
+            first_note_lower = first_note_raw.lower()
+
+            if first_note_lower.startswith("forfeited by"):
+                info.status = "completed"
+                lines = [l.strip() for l in first_note_raw.split("\n") if l.strip()]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)